src/helpers/matchers/matchTextContent.ts

```diff
@@ -7,9 +7,10 @@ import { getTextContent } from '../getTextContent';
 export function matchTextContent(
   node: ReactTestInstance,
   text: TextMatch,
+  typeToMatch: ElementType | string,
   options: TextMatchOptions = {}
```

Comment:
we could maybe use an object for the params here wdyt?

Reply:
Yes absolutely